pyproject.toml

```diff
@@ -37,6 +37,7 @@ requires = [
     "httpx[http2]==0.23.0",
     "authlib==1.0.1",
     "pydantic>=1.8.0",
+    "strawberry-graphql==0.171.1",
```

Comment:
Is this exact-version pinning necessary? In libraries you generally want to avoid exact pinning so clients if the library can use their applications with a potentially different version. This creates potential problems with conflicting versions for users of the lib.

Reply:
nice catch, will change it, no reason for the exact pinning.

Reply:
```suggestion
    "strawberry-graphql>=0.171.1",
```